src/qt/numeric-formats/qml/TimecodeControl.qml

```diff
@@ -0,0 +1,348 @@
+import QtQuick
```

Comment:
Generally, it's not a TimecodeControl, it's really a NumericTextControl. It is suitable right now to display Time, Duration, Frequency and Bandwith